fw/RadioTask.c

```diff
@@ -1279,7 +1284,10 @@ static void reactToAdvExtPDU(const BLE_Frame *frame, uint8_t advLen)
         else if (phy == PHY_2M)
             auxPeriod = (AUX_OFF_TARG_USEC + 2000) * 4;
         else // (phy == PHY_CODED_S8 || phy == PHY_CODED_S2)
-            auxPeriod = (AUX_OFF_TARG_USEC + 20000) * 4;
+            // PHY_CODED_S2:  6534us =  4542 + 150 + 1006 + 150 +  686
+            // PHY_CODED_S8: 21852us = 17040 + 150 + 2896 + 150 + 1616
+            // 22000 >= 21852
+            auxPeriod = (AUX_OFF_TARG_USEC + 22000) * 4;
```

Comment:
Good note here, I might make a patch related to this and credit you

Reply:
These timings are the longest possible advertisement, plus an AUX_CONNECT_REQ, plus an AUX_CONNECT_RSP?